near-sdk-macros/src/core_impl/code_generator/impl_item_method_info.rs

```diff
@@ -136,6 +138,30 @@ impl ImplItemMethodInfo {
                     #method_invocation;
                     #contract_ser
                 },
+                ReturnType::Type(_, return_type)
+                    if utils::type_is_result(return_type) && *is_returns_result =>
+                {
+                    let value_ser = match result_serializer {
+                        SerializerType::JSON => quote! {
+                            let result = near_sdk::serde_json::to_vec(&result).expect("Failed to serialize the return value using JSON.");
```

Comment:
I see that we're making similar calls to `expect` in a couple places, but @austinabell do we want to eventually get rid of these `expect` calls since they should be bringing the formatting mechanism. Or is there some reason we're keeping them?

Reply:
iirc this expect doesn't embed file-specific data in the error message when used in a proc macro, but I could be remembering wrong why I didn't switch these.

Might be worth opening an issue and double-checking before next release, but wouldn't be a breaking change to come in after anyway

Reply:
ticket: https://github.com/near/near-sdk-rs/issues/746